### PR TITLE
Update docs and use iodata

### PR DESCRIPTION
**Pull request by jonatanklosko**
🐈

### CODE REVIEW

**Review by josevalim — APPROVED** [review]

File: mix.exs [hunk 1]
```diff
@@ -1,41 +1,48 @@
 defmodule Safetensors.MixProject do
   use Mix.Project
 
-  @description "Safetensors dump and load in Elixir"
+  @version "0.1.1"
```
Comment (jonatanklosko) [inline mix.exs:4]: @grzuy I don't think we need 0.2.0, 0.1.0 only has load and we are not breaking it, right?

Comment (grzuy) [reply mix.exs:4]: Hi @jonatanklosko,

Either works for me.
My interpretation from semantic versioning has always been anything 0.x could be potentially be considered breaking, so when updating 0.x branches I tend to update minor (nor tiny) digit. Using tiny also seems fine :+1: 

Anyway, I already published 0.2.0 a few minutes before you opened this PR he :-)
You guys good or prefer we retire?

**Comment by grzuy** [comment]
Nice!

File: mix.exs (hunk 1 above)
Comment (jonatanklosko) [reply mix.exs:4]: Oh, I changed `decode/1` to return iodata and not binary, so it is a breaking change, we would need 0.3 right away. You can unpublish the package version within 60 minutes, so I would just do that :)

Comment (jonatanklosko) [reply mix.exs:4]: > My interpretation from semantic versioning has always been anything 0.x could be potentially be considered breaking

Right, it's just that I think there was no breaking change to `load/1` from 0.1.0, only the addition of `decode/1`, so `0.1.1` should be fine.

(not that it matters much here given people don't depend on the package already :))

Comment (grzuy) [reply mix.exs:4]:
```
Reverted safetensors 0.2.0
```

Comment (jonatanklosko) [reply mix.exs:4]: Thanks! I will do #2 and publish afterwards :)

Comment (grzuy) [reply mix.exs:4]: Awesome!

**Pull request merged**